packages-ts/integration-eqlabs-multisig/test/Multisig.test.ts

```diff
@@ -3,10 +3,14 @@ import { starknet } from 'hardhat'
 import { StarknetContract, Account } from 'hardhat/types/runtime'
 import { getSelectorFromName } from 'starknet/dist/utils/hash'
 import { number } from 'starknet'
+import { account } from '@chainlink/starknet'
 
 describe('Multisig integration tests', function () {
   this.timeout(300_000)
 
+  const opts = account.makeFunderOptsFromEnv()
+  const funder = account.Funder(opts)
```

Comment:
Looks like the test is failing b/c of this:

```suggestion
  const funder = new account.Funder(opts)
```